esphome/core/defines.h

```diff
@@ -38,6 +38,7 @@
 #define USE_TEXT_SENSOR
 #define USE_TIME
 #define USE_WEBSERVER
+#define USE_UART_DEBUGGER
```

Comment:
Alphabetical order =)

```suggestion
#define USE_UART_DEBUGGER
#define USE_WEBSERVER
```